tests/integration/components/setup-global-a11y-hooks-test-test.ts

```diff
@@ -0,0 +1,52 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import {
+  setupGlobalA11yHooks,
+  teardownGlobalA11yHooks,
+  setEnableA11yAudit,
+} from 'ember-a11y-testing/test-support';
+import { render, click } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+
+module(
+  'Integration | Component | setup-global-a11y-hooks-test',
+  function (hooks) {
+    setupRenderingTest(hooks);
+
+    let actualAuditInvocationsCount = 0;
+
+    function invokeAll(): boolean {
+      return true;
+    }
+
+    async function a11yAuditFake() {
```

Comment:
I'd prefer we either convert https://github.com/ember-a11y/ember-a11y-testing/blob/master/tests/acceptance/setup-global-a11y-hooks-test.ts#L36 to an `async` function, or we don't use `async` here and return a resolved promise like the existing test is doing. This is mainly for consistency between these very similar tests.